Lib/selectors.py

```diff
@@ -338,92 +338,90 @@ def select(self, timeout=None):
         return ready
 
 
-if hasattr(select, 'poll'):
+class _PollLikeSelector(_BaseSelectorImpl):
+    """Base class shared between poll, epoll and devpoll selectors."""
 
-    class PollSelector(_BaseSelectorImpl):
-        """Poll-based selector."""
+    def __init__(self):
+        super().__init__()
+        self._poller = self._poller()
```

Comment:
How would you feel about having a `_selector` attribute for all classes (also kqueue and select)?